pkg/convert/go_messages.go

```diff
@@ -0,0 +1,28 @@
+package convert
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/expect-digital/translate/pkg/model"
+)
+
+func FromGoMessages(m model.Messages) ([]byte, error) {
+	msg, err := json.Marshal(m)
```

Comment:
We require to transform `model.Messages` into `pipelines.Messages` before JSON encoding. `model.Messages`.

Reply:
Aaaa I get it ;D

Reply:
I think i done, but not really sure. Check when you will have time ;)